_data/simple-icons.json

```diff
@@ -10992,6 +10992,11 @@
             "hex": "7D00FF",
             "source": "https://www.stellar.org/press"
         },
+        {
+            "title": "Stencil",
+            "hex": "000000",
+            "source": "https://cdn.svgporn.com/logos/stenciljs-icon.svg"
```

Comment:
Did not review the icon yet, but this is not an official source.

Reply:
It wold be better to use the SVG from the Website that can be found at https://github.com/ionic-team/stencil-site/tree/main/static/img